development/build/etc.js

```diff
@@ -45,7 +46,9 @@ function createZipTask(platform, buildType, version) {
         : `metamask-${buildType}-${platform}-${version}`;
     await pump(
       gulp.src(`dist/${platform}/**`),
-      gulpZip(`${path}.zip`),
+      // sort files and set `mtime` to epoch to ensure zip build is deterministic
+      sort(),
+      gulpZip(`${path}.zip`, { modifiedTime: new Date(0) }),
```

Comment:
Why can't we use the current unix timestamp? I.e. `Date.now()`?

Reply:
Because the `gulp-zip` accepts a `Date` object for the `modifiedTime` option: https://www.npmjs.com/package/gulp-zip

Reply:
That would make the build non-deterministic. It would be different each time.

Reply:
Sorry I guess that's not true, they would be "deterministic" for a known timestamp. They would not be reproducible though. Reproducible is the word I should have used. That is the requirement this is trying to meet.